src/rofiles-fuse/main.c

```diff
@@ -438,14 +438,14 @@ callback_access (const char *path, int mode)
 
 static int
 callback_setxattr (const char *path, const char *name, const char *value,
-		   size_t size, int flags)
+                   size_t size, int flags)
 {
   return -ENOTSUP;
 }
 
 static int
 callback_getxattr (const char *path, const char *name, char *value,
-		   size_t size)
+       size_t size)
```

Comment:
Done, I also ran it through the Emacs `indent-region` while here, since it was almost clean.